core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -210,9 +210,17 @@ class KafkaServer(val config: KafkaConfig, time: Time = Time.SYSTEM, threadNameP
         _clusterId = getOrGenerateClusterId(zkClient)
         info(s"Cluster ID = $clusterId")
 
+        /* load metadata */
+        val (brokerMetadata, initialOfflineDirs) = getBrokerMetadataAndOfflineDirs
```

Comment:
nit: To avoid confusion, I wonder if we could rename `brokerMetadata` to `preloadedBrokerMetadata` or something like that?

Reply:
Done.